app/liquid_tags/runkit_tag.rb

```diff
@@ -66,13 +66,9 @@ def self.script
   end
 
   def sanitized_preamble(markup)
-    raise StandardError, "Runkit tag is invalid" if markup.ends_with? "\">"
+    raise StandardError, "Runkit tag is invalid" if markup.include? "\""
```

Comment:
This line prevents the use of double quotes, not the use of `>` characters. This breaks all runkit tags that use double-quoted strings. For example, the following is now a minimal example of an invalid runkit tag:

```
{% runkit "" %}
{% endrunkit %}
```

Whereas this is fine:

```
{% runkit '' %}
{% endrunkit %}
```

This is probably related to #1230 and #1228.